### ISSUE TITLE
texturify own mesh

### ISSUE THREAD

**Issue by xxplyy** [TEXTurePaper/TEXTurePaper#53]
Thanks for the excellent work,but I still have some questions here.Is there a simple way to use this method to texturify my own 3D mesh?Because I found the .yaml file is hard for me to get through.What does the "seed" mean anyway?Using the pretrained model is enough to me,just like how napoleon and nascar be painted, and I don't have enough resourses to train a fine-tuned model.
